### PR TITLE
Abzu 47924 Initial NM load script

### PR DESCRIPTION
**Pull request by barrie-cooper**
LINQPad script to load public Daily & Weekly NM files.

### CODE REVIEW

File: Tools/LoadNMs.linq [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+
+	var fssClient = new FileShareApiAdminClient(new UserAgentClientFactory(), config.BaseAddress,
+		new AuthTokenProvider(config.ClientId, config.TenantId, config.AuthUrl));
+
+	if (REPLACE_ALL_DATA_IN_BUSINESS_UNIT)
+	{
+		await ExpireAllDataInBusinessUnit(fssClient, BUSINESS_UNIT);
+	}
+
+	await ProcessDailies(fssClient, @"\\hadsprddom04.business.ukho.gov.uk\DFS\Prod_Prod\nms\FILES\NMOutput\Dailies");
```
Comment (crdandoukho) [inline Tools/LoadNMs.linq:62]: I would prefer that these internal paths were not visible in a public repo

File: Tools/LoadNMs.linq [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+private readonly Func<DateTime, DateTime> DAILY_NMS_RETENTION_PERIOD = d => d.AddDays(15);
+
+private readonly bool REPLACE_ALL_DATA_IN_BUSINESS_UNIT = false;
+
+private async Task Main()
+{
+	var configLive = new
+	{
+		BaseAddress = @"https://files.admiralty.co.uk/",
+		ClientId = "30cc156d-765f-43ea-8c84-a97d5b8ce852",
```
Comment (crdandoukho) [inline Tools/LoadNMs.linq:33]: I personally do not feel comfortable making these details public. Fair enough if these were B2C app registrations that are already in the public domain, but these are from the internal domain.

**Review by crdandoukho — CHANGES_REQUESTED** [review]

File: Tools/LoadNMs.linq (hunk 1 above)
Comment (barrie-cooper) [reply Tools/LoadNMs.linq:62]: I completely agree, so have deferred to storing this file in the "Support Engineer Toolbox".

Please see https://github.com/UKHO/supportengineer-toolbox/pull/62

File: Tools/LoadNMs.linq (hunk 2 above)
Comment (barrie-cooper) [reply Tools/LoadNMs.linq:33]: I completely agree, so have deferred to storing this file in the "Support Engineer Toolbox".

Please see https://github.com/UKHO/supportengineer-toolbox/pull/62

**Pull request closed without merging**